### PR TITLE
Add community-hotline.lang

### PR DESCRIPTION
**Pull request by craigcook**
Lang file added in https://github.com/mozilla-l10n/www.mozilla.org/pull/362

### CODE REVIEW

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -416,6 +416,11 @@
             'de', 'es-ES', 'fr', 'hi-IN', 'ja', 'pt-BR', 'zh-TW',
         ],
     ],
+    'mozorg/about/governance/policies/community-hotline.lang' => [
+        'supported_locales' => [
+            'de', 'es-ES', 'fr', 'hi-IN', 'ja', 'pt-BR', 'zh-TW',
```
Comment (peiying2) [inline app/config/sources.inc.php]: per bug https://bugzilla.mozilla.org/show_bug.cgi?id=1600148#c8 - the locale list should expand to 

 ar, de, es-ES, fr,  hi, id, it, ja, nl, pl, pt-BR, ru,  zh-CN, zh-TW

This means all three CPG files need to be updated with the locale list

Comment (craigcook) [reply app/config/sources.inc.php]: Ah, I missed that. Thanks!

**Comment by peiying2** [comment]
Let's change `hi` to `hi-IN`. On mozilla.org, we only have hi-IN.

File: app/config/sources.inc.php [hunk 2]
```diff
@@ -408,12 +408,17 @@
     ],
     'mozorg/about/governance/policies/participation.lang' => [
         'supported_locales' => [
-            'de', 'es-ES', 'fr', 'hi-IN', 'ja', 'pt-BR', 'zh-TW',
+            'ar', 'de', 'es-ES', 'fr', 'hi', 'id', 'it', 'ja', 'nl', 'pl', 'pt-BR', 'ru', 'zh-CN', 'zh-TW',
```
Comment (flodolo) [inline app/config/sources.inc.php]: These pages are not opt-in, and I assume the list of locales is always going to be the same for all of them?

If that's the case, to make this less error-prone, you can also define an array at the top of the file, and only reference the array here ([example](https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L29)).

Comment (peiying2) [reply app/config/sources.inc.php]: That's true, the same locales for all three pages

Comment (craigcook) [reply app/config/sources.inc.php]: Updated!

File: app/config/sources.inc.php [hunk 3]
```diff
@@ -407,14 +412,13 @@
...
-        ],
+        'supported_locales' => $participation_locales,
     ],
     'mozorg/about/governance/policies/reporting.lang' => [
-        'supported_locales' => [
-            'de', 'es-ES', 'fr', 'hi-IN', 'ja', 'pt-BR', 'zh-TW',
-        ],
+        'supported_locales' => $participation_locales,
+    ],
+    'mozorg/about/governance/policies/community-hotline.lang' => [
```
Comment (peiying2) [inline app/config/sources.inc.php]: nit pick: let's make sure it is in alphabetic order.

**Pull request merged**